CHANGELOG.md

```diff
@@ -4,30 +4,37 @@
 
 ## DFX
 
+### feat: updates the dfx new starter project for env vars
+
+- Updates the starter project for env vars to use the new `dfx build` & `dfx deploy` environment variables
+- Changes the format of the canister id env vars to be `<canister_name>_CANISTER_ID`, matching the existing convention
```

Comment:
Matching the existing convention where?

Reply:
From the `process_string` we insert into `index.js.hbs`. https://sourcegraph.com/-/editor?remote_url=git%40github.com%3Adfinity%2Fsdk.git&branch=kyle%2Frefactor-starter-for-env&file=src%2Fdfx%2Fsrc%2Flib%2Fbuilders%2Fmod.rs&editor=VSCode&version=2.2.14&start_row=276&start_col=13&end_row=276&end_col=13